cypress/page/login.page.ts

```diff
@@ -0,0 +1,25 @@
+class LoginPage {
+  private emailInput: string;
+  private passwordInput: string;
+  private SignInBtn: string;
+
+  constructor() {
+    this.emailInput = "#email";
+    this.passwordInput = "#passwd";
+    this.SignInBtn = "[name='SubmitLogin']";
+  }
+
+  public typeEmailInput(): void {
+    cy.get(this.emailInput).type("aperdomobo@gmail.com");
+  }
+
+  public typePasswordInput(): void {
+    cy.get(this.passwordInput).type("WorkshopProtractor");
+  }
+
+  public clickSignInBtn(): void {
+    cy.get(this.SignInBtn).click();
+  }
+}
+
+export default new LoginPage();
```

Comment:
not sure about this approach, I prefer the one suggested by the guide, export the class and then have a page/index.ts so you can import from the page index in the buy-shirt.cy.ts file